test/unstable-keyword/deprecated/1.30/noParens-detailed.good

```diff
@@ -3,6 +3,6 @@
   For this attribute:
       |
     1 | @unstable "foo is unstable and may change"
-      |            ⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺
+      |           ⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺
```

Comment:
I can't understand why the github diff doesn't look like it has the right number of dashes here; but looking at the .good file, it seems correct to add one more on the left.